### PR TITLE
Adding mellii

### PR DESCRIPTION
**Pull request by mellii**
@unnderbar merge please :)

### CODE REVIEW

File: _hackers/mellii.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+fullname: mellii
+twitter: 
+image: mellii.jpg
```
Comment (tobiasahlin) [inline _hackers/mellii.md]: @mellii Looks like this doesn't match the image filename (.jpg vs .png) :astonished:

Comment (mellii) [inline _hackers/mellii.md]: oops :)

Comment (mellii) [inline _hackers/mellii.md]: @tobiasahlin fixed. try again?

**Comment by tobiasahlin** [comment]
:shipit:

**Pull request merged**